hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/http/TestSSLHttpServer.java

```diff
@@ -291,13 +294,30 @@ public void testExcludedCiphers() throws Exception {
   @Test
   public void testIncludedProtocols() throws Exception {
     URL url = new URL(baseUrl, SERVLET_PATH_ECHO + "?a=b&c=d");
+
+    String includedProtocols;
+    String version = System.getProperty("java.version");
+    if (version.startsWith("1.")) { // JDK 8
+      includedProtocols = INCLUDED_PROTOCOLS;
+    } else { // JDK 11 and above
+      includedProtocols = INCLUDED_PROTOCOLS_JDK11;
+    }
```

Comment:
We can use the utility method and don't have to parse the java.version string. This change can avoid test failure in Java 9 and 10 (corner case).
```suggestion
    if (Shell.isJavaVersionAtLeast(11) {
      includedProtocols = INCLUDED_PROTOCOLS_JDK11;
    } else {
      includedProtocols = INCLUDED_PROTOCOLS;
    }
```